erpnext/hr/doctype/expense_receipt/expense_receipt.json

```diff
@@ -0,0 +1,536 @@
+{
```

Comment:
Fix the layout of the form, important and mandatory fields will be on first section and on left if possible. In my view, it should be like this:

1st sec-left:

1. employee
2. expense type
3. total receipt amount
4. total tax

1st sec-right:
1. expense date
2. Is claimed

Other things can go to additional details